### PR TITLE
Add docs on how to integrate with dynamo

### PR DESCRIPTION
**Pull request by changm**

### CODE REVIEW

File: TROUBLESHOOTING.md [hunk 1]
```diff
@@ -58,6 +58,10 @@ report sent to us if you have it.
 
 You can enable the PyTorch/XLA debugging tool by setting `PT_XLA_DEBUG=1`, which provides a couple useful debugging features.
 
+## PyTorch/XLA + Dynamo Debugging Tool
+
+You can enable the PyTorch/XLA + Dynamo debugging tool by setting `XLA_DYNAMO_DEBUG=1`, which provides a couple useful debugging features.
```
Comment (cota) [inline TROUBLESHOOTING.md:63]: (Nit-picking)
I see that this is copied from above, but without mentioning those "couple [of] useful debugging features", the statement is redundant.
I suggest we drop everything after the comma (`, which provides...`).

Comment (cota) [reply TROUBLESHOOTING.md:63]: Oh I can add a suggestion, let me do that.

Comment (cota) [inline TROUBLESHOOTING.md:63]:
```suggestion
You can enable the PyTorch/XLA + Dynamo debugging tool by setting `XLA_DYNAMO_DEBUG=1`.
```

File: docs/dynamo.md [hunk 2]
```diff
@@ -40,7 +63,7 @@ timm_vision_transformer | 3.52
 geomean | 3.04
 
 Note 
-1. User will likely see better inference perfomrance by putting the inference execution in a `torch.no_grad` context. `openxla` is a `aot-autograd` backend of `torch.compile`. `Aot-autograd` will attempt to save some states for potential backward. `torch.no_grad` will help `aot-autograd` understand that it is being executed in a inference context.
+1. User will likely see better inference performance by putting the inference execution in a `torch.no_grad` context. `openxla` is a `aot-autograd` backend of `torch.compile`. `Aot-autograd` will attempt to save some state for potential backward. `torch.no_grad` will help `aot-autograd` understand that it is being executed in a inference context.
```
Comment (cota) [inline docs/dynamo.md:66]:
```suggestion
1. User will likely see better inference performance by putting the inference execution in a `torch.no_grad` context. `openxla` is an `aot-autograd` backend of `torch.compile`; `aot-autograd` attempts to save some state for a potential backward pass. Setting `torch.no_grad` helps `aot-autograd` understand that it is being executed in an inference context.
```

**Comment by cota** [comment]
Thanks for this! Just left some small suggestions.

Note: Not necessary since this is just a doc change, but in case you're wondering: in order to get CI not to time out, you would have to push this change to a branch on the main repo -- the build cache is enabled there, which speeds up CI builds considerably.

**Comment by changm** [comment]
Ahh thanks for the heads up. Yeah I tried another PR that committed to master and that seemed to work. I also committed your changes here, let me know if there's anything else you'd like to change.

**Review by cota — APPROVED** [review]

**Pull request merged**